evento/src/test-helpers/mock-factory.js

```diff
@@ -6,6 +6,13 @@ const generateId = () =>
 const generateString = (n) =>
 	(Math.random().toString(36) + '00000000000000000').slice(2, n + 2);
 
+// Return time stamp of day after tomorrow
+const generateTimeStamp = () => {
```

Comment:
Should the name be something else or get a (defaulted) parameter for how many days from now?